node/bft/events/src/block_response.rs

```diff
@@ -37,25 +49,62 @@ impl<N: Network> EventTrait for BlockResponse<N> {
     }
 }
 
+impl<N: Network> BlockResponse<N> {
+    /// The version number that we use to serialize block responses,
+    /// and the highest known message valid version number.
+    const CURRENT_VERSION_NUMBER: u8 = 2;
+    /// The minimum supported message version number.
+    const MIN_VERSION_NUMBER: u8 = 1;
```

Comment:
When do these two values get updated? Is it every time we have a new consensus version or message version?

If so, we should probably have a test of some sort to enforce we are updating all the versions in tandem for each release.

Reply:
My hope is that we can bump the minimum to `2`  and drop support for responses that don't include the consensus version in the near future. Aside from that, changes to it should be fairly infrequent.

Reply:
Done in [20c3dd8](https://github.com/ProvableHQ/snarkOS/pull/3936/commits/20c3dd854d787b58b2fd1df97473e910453878aa).